lib/bundler/plugin/index.rb

```diff
@@ -26,6 +25,7 @@ def initialize
         @sources = {}
         @hooks = {}
         @load_paths = {}
+        @installed_plugins = []
```

Comment:
`@plugin_paths.keys` shall handle this. I don't think there is any need to explicitly store it.